rcon/watch_killrate.py

```diff
@@ -0,0 +1,389 @@
+"""
+    Ported with some tweaks/additions from https://github.com/ElGuillermo/HLL_CRCON_Discord_watch_killrate/blob/main/hll_rcon_tool/custom_tools/watch_killrate.py
+"""
+
+from logging import getLogger
+from rcon.api_commands import get_rcon_api, RconAPI
+from rcon.types import GetDetailedPlayer
+from collections import Counter
+from rcon.user_config.watch_killrate import WatchKillRateUserConfig
+from rcon.rcon import SERVER_INFO
+from datetime import datetime
+from collections import defaultdict
+import sys
+from datetime import datetime, timedelta
+from time import sleep
+import discord
+from rcon.user_config.rcon_server_settings import RconServerSettingsUserConfig
+from rcon.scoreboard import current_game_stats
+
+logger = getLogger(__name__)
+
+# TODO: move these to a common location and get translations from T17
+# sourced from https://gist.github.com/timraay/5634d85eab552b5dfafb9fd61273dc52#playerinfo-player_name
+ARTILLERY: dict[str, bool] = {
```

Comment:
With the weapons.py file from https://github.com/MarechJ/hll_rcon_tool/pull/797 this could be removed. We should, no matter which one we keep, not have to maintain two different sets of Weapon lists in CRCon 😅

Reply:
Yeah I very much agree; do you want to approve for now and then I will update once that gets merged?

I have asked T17 (and just bumped the request) to get translations for map names/weapon names for all the versions they've localized for, and once we have those I'd like to add some options to localize that in places where it gets displayed to the user (like chat commands and so on)